assets/translations/en-us.yaml

```diff
@@ -6156,6 +6156,7 @@ epinio:
     name: Name
     create: Create a Namespace
   services:
+    pairs: Configure data that will be available to a container
```

Comment:
I think this is better as `Environment Variables` with a `<i v-tooltip="'Environment variables are available to bound applications'" class="icon icon-info" />`

Reply:
I got this wrong, it should still reference data